tests/memgraph/test_query_builder.py

```diff
@@ -1232,6 +1245,26 @@ def test_base_class_with(memgraph):
     mock.assert_called_with(expected_query)
 
 
+def test_base_class_load_csv(memgraph):
+    query_builder = load_csv("path/to/my/file.csv", True, "row").return_()
+    expected_query = " LOAD CSV FROM 'path/to/my/file.csv' WITH HEADER AS row RETURN * "
+
+    with patch.object(Memgraph, "execute_and_fetch", return_value=None) as mock:
+        query_builder.execute()
+
+    mock.assert_called_with(expected_query)
+
+
+def test_base_class_return(memgraph):
+    query_builder = return_({"n": "n"})
```

Comment:
Now I started thinking about this and I returned to the `dict_to_alias_statement` method in the query builder. As I saw there, this is the only way to create `RETURN n` statement and I was surprised. Do you think it would be better if `results` in the `RETURN` clause could be either a simple `string` or `Dict` of strings? Then if it's a simple string, we just do `RETURN n`, and if it's not, the check stays the same as it is now.

Reply:
Here is the issue for this: https://github.com/memgraph/gqlalchemy/issues/140